File: BEE 1011.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const pi = 3.14159;
var r = parseFloat(lines);
var ans = (4.0/3)*pi*(r*r*r);

console.log("VOLUME =", ans.toFixed(3));

File: BEE 1011.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        double pi = 3.14159;
        double r = input.nextDouble();
        double ans = (4.0/3)*pi*(r*r*r);
        System.out.printf("VOLUME = %.3f\n", ans);
    }
}